### ISSUE TITLE
[Tiny sur Mer] the bugs and the non-bugs

### ISSUE THREAD

**Issue by Obsidien** [agateau/pixelwheels#48]
In Tiny-sur-Mer [French typography requires the hyphens, normally], there are real bugs, false bugs, hard to say bugs, and other.

First, after the first curve, there’s a road that goes outside the circuit, that is not blocked by fences. Similarly, there’s a small space at the second curve.

Another one: after the wood bridge, there’s a sand space that is not part of the circuit (the second sand space), where you can go by following the sea; but, there’s also one of the fences that do not block the car, is that wanted?

Also, from the previous sand space, you can, following the sea, go in a parking (?); there’re various bugs there, you’re blocked by a house that does not exist, and you can go through one that exists; and finally, following the sea, you’re blocked by the shadow of the third bridge…

One more bug: when going backward, you can traverse the sea, you’re not stopped like when you’re advancing, you just need to finish before the helicopter comes.

Well… there’s also the shorpath (by killing you near the shadow of the first bridge). Or should I say the shortpaths? (one by the sea, and one by traversing a house.) Are there really wanted? I find that a bit too much. Allowing going there, why not (there’s just the problem of being blocked by the bridge shadow, again), but you should not be able to win a lap by killing you.

**Comment by Wilker-a** [comment]
the part of traversing the sea is not that bad. tha deeper water reduces the speed by a lot, otherwise it would actually become a problem.

**Comment by Obsidien** [comment]
> the part of traversing the sea is not that bad. tha deeper water reduces the speed by a lot, otherwise it would actually become a problem.

I think you missed my point, or I’m missing yours. I regularly go through the sea a bit, and know that my car have its speed reduced until it stops (or I go out); all that’s working great. Problem is, when going forward, you’re (reasonably) rapidly stopped, while when going backward, that doesn’t oddly apply. That’s a problem of coherence of the game, and one that might allow some unintended shortpaths on future maps if not solved.

**Comment by Wilker-a** [comment]
i in fact misread and missed the going backwards part, sorry for that \^\^'

**Comment by agateau** [comment]
Nice catches!
I just fixed the map-definition related bugs. I am going to close this one open a separate one for the "going backward in the sea does not slow the vehicle down" since it's not a map issue, it just only happens on Tiny-sur-mer because it's the only one with water for now.
